subsys/bluetooth/mesh/adv_ext.c

```diff
@@ -482,7 +482,7 @@ int bt_mesh_adv_enable(void)
 			return err;
 		}
 
-		if (IS_ENABLED(CONFIG_BT_LL_SOFTDEVICE) && adv->tag & BT_MESH_FRIEND_ADV) {
+		if (IS_ENABLED(CONFIG_BT_LL_SOFTDEVICE) && adv->tag == BT_MESH_FRIEND_ADV) {
```

Comment:
Will it be better to use `IS_ENABLED(CONFIG_BT_MESH_ADV_EXT_FRIEND_SEPARATE)` instead to let compiler explicitly know it is possible to cut out?

Reply:
Sure 👍